src/ax/marshaling/message_test.go

```diff
@@ -32,37 +34,64 @@ var _ = Describe("MarshalMessage", func() {
 })
 
 var _ = Describe("UnmarshalMessage", func() {
+
 	message := &messagetest.Message{
 		Value: "<value>",
 	}
-	data, err := proto.Marshal(message)
+
+	pbdata, err := proto.Marshal(message)
+	if err != nil {
+		panic(err)
+	}
+
+	_, jsondata, err := MarshalJSON(message)
 	if err != nil {
 		panic(err)
 	}
 
 	It("unmarshals the message using the protocol specified in the content-type", func() {
```

Comment:
I think this pair of tests would be a good candidate for moving to a table-driven test using the ginkgo `table` package. It can then easily be extended when CBOR support is added.